### PR TITLE
Adding in actions/filters that are applicable to build ontop of anvato

### PR DESCRIPTION
**Pull request by srtfisher**
Backported updates from where we are building on top of Anvato.

### CODE REVIEW

File: lib/class-anvato-settings.php [hunk 1]
```diff
@@ -59,8 +59,8 @@ public function action_admin_init() {
...
 		add_settings_field( 'width', __( 'Default Width:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'width' ) );
 		add_settings_field( 'height', __( 'Default Height:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'height' ) );
-		add_settings_field( 'public_key', __( 'Public Key:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'public_key' ) );
-		add_settings_field( 'private_key', __( 'Private Key:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'private_key', 'type' => 'password' ) );
+		add_settings_field( 'public_key', __( 'Station Public Key:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'public_key' ) );
+		add_settings_field( 'private_key', __( 'Station Private Key:', 'anvato' ), array( self::$instance, 'field' ), self::SLUG, 'general', array( 'field' => 'private_key', 'type' => 'password' ) );
```
Comment (dlh01) [inline lib/class-anvato-settings.php]: I think it got buried in our other discussion, but I don't think we can be sure "Station" is the word to use here.

File: lib/class-anvato-library.php [hunk 2]
```diff
@@ -180,7 +203,8 @@ private function request( $params ) {
 		$url = $this->build_request_url( $params, time() );
 		$args = array( 'body' => $this->xml_body );
 		if ( function_exists( 'vip_safe_wp_remote_get' ) ) {
-			$response = vip_safe_wp_remote_get( $url, false, 3, 2, 20, $args );
+			// Timeout increased for the master branch only.
+			$response = vip_safe_wp_remote_get( $url, false, 3, 3, 20, $args );
```
Comment (dlh01) [inline lib/class-anvato-library.php]: This change doesn't belong upstream.

**Comment by dlh01** [comment]
One item to fix, one for your consideration, otherwise :guitar:

File: lib/class-anvato-settings.php (hunk 1 above)
Comment (srtfisher) [inline lib/class-anvato-settings.php]: Only reasoning was the use of 'Station ID' a few lines up, I'll remove 'Station'

**Pull request merged**